handlers/main.yml

```diff
@@ -1,6 +1,7 @@
 # handlers file for supervisor
 ---
 - name: restart supervisor
+  become: yes
```

Comment:
`become` does not work in Ansible < 1.9. Can't this be specified at playbook level?